### ISSUE TITLE
Protect users from accessing `/adminDashboard`

### ISSUE THREAD

**Issue by Caleb-Cohen** [Together-100Devs/Together#439]
## Please provide a summary of the feature
Currently admin dashboard can be access using React Router by appending `/adminDashboard` to the end. We want to prevent users from access adminDashboard. Ideally, it blocks the user on request instead of redirecting them once they land on the page.

## Depends on
#441 

## Is this feature related to any current or past issues? Please provide the
- #423 
- #407 

## Additional context
This is a tough cookie. React router seems a little restrictive with our options considering the current app design. 

_If you're interested in working on this feature, please comment so you can be assigned_

**Comment by cblanken** [comment]
I'd like to take another look at this, but if I haven't gotten anywhere by the weekend, someone should feel free to pick it up.

**Comment by r-Dev03** [comment]
When you say "block them" what are you exactly referring to? Do you mean send them to a page that notifies them that only admin's can access that page? That would be considered some form of "blocking"

**Comment by Caleb-Cohen** [comment]
> When you say "block them" what are you exactly referring to? Do you mean send them to a page that notifies them that only admin's can access that page? That would be considered some form of "blocking"

I've kept it vague as I'm not familiar with all our options. Ideally, un-authorized users will not have access to the `/Admindashboard` page. Any attempt to access will redirect back to homepage, without accessing any part of the page.

**Comment by r-Dev03** [comment]
Hmm, sounds good, I can take a crack at it if don't mind assigning me

**Comment by Caleb-Cohen** [comment]
> Hmm, sounds good, I can take a crack at it if don't mind assigning me

assigned!

**Comment by Caleb-Cohen** [comment]
On hold for #441

**Comment by DevinCLane** [comment]
closing as stale
